ci-operator/step-registry/ipi/conf/vsphere/template/ipi-conf-vsphere-template-commands.sh

```diff
@@ -42,17 +37,27 @@ else
   vm_template="${RHCOS_VM_TEMPLATE}"
 fi
 
-cat >> "${SHARED_DIR}"/enable_template_content.txt << EOF
+cat > "${PATCH}" << EOF
+platform:
+  vsphere:
     failureDomains:
     - name: generated-failure-domain
```

Comment:
I guess we even can remove the `name` line in this patch file.

Reply:
I tried this and seems it will replace other lines.
like that:
platform:
  vsphere:
    apiVIPs:
    - xxxx
    failureDomains:
      topology:
        template: /${vsphere_datacenter}/vm/${vm_template}
 it should be :
 platform:
  vsphere:
    apiVIPs:
    - xxxxxx
    failureDomains:
    - name: generated-failure-domain
      region: generated-region
      server: vcenter.devqe.ibmc.devcluster.openshift.com
      topology:
        computeCluster: /DEVQEdatacenter/host/DEVQEcluster
        datacenter: DEVQEdatacenter
        datastore: /DEVQEdatacenter/datastore/vsanDatastore
        template: /${vsphere_datacenter}/vm/${vm_template}

Reply:
what was the patch yaml content? I tried it locally it works.
```
$ cat test1.yaml
platform:
  vsphere:
    apiVIPs:
    - xxxxxx
    failureDomains:
    - name: generated-failure-domain
      region: generated-region
      server: vcenter.devqe.ibmc.devcluster.openshift.com
      topology:
        computeCluster: /DEVQEdatacenter/host/DEVQEcluster
$ cat test1.yaml.patch
platform:
  vsphere:
    failureDomains:
    - topology:
        template: /${vsphere_datacenter}/vm/${vm_template}
$ yq-3.3.0 m -x -i test1.yaml test1.yaml.patch
$ cat test1.yaml
platform:
  vsphere:
    apiVIPs:
    - xxxxxx
    failureDomains:
    - name: generated-failure-domain
      region: generated-region
      server: vcenter.devqe.ibmc.devcluster.openshift.com
      topology:
        computeCluster: /DEVQEdatacenter/host/DEVQEcluster
        template: /${vsphere_datacenter}/vm/${vm_template}

```

Reply:
ok. thanks, my topology field missing "-".